compiler/rustc_ast_lowering/src/local.rs

```diff
@@ -0,0 +1,68 @@
+use super::LoweringContext;
```

Comment:
A `local.rs` file seems a bit too restrictive. Should this be a `stmt.rs` file with all code lowering stmt/block/local?

Reply:
How about `block.rs` with block, stmt, local?